jetson/odrive_bridge/src/__main__.py

```diff
@@ -287,16 +287,16 @@ def get_state(self):
 """
 
 
-def publish_state_msg(msg, state):
-    global legal_controller
-    msg.state = states.index(state)
-    msg.controller = legal_controller
+def publish_state_msg(msg, state_string):
+    global odrive_controller_index
+    msg.state = state_string[:len(state_string) - len("_State")]

```

Comment:
assign :len(state_string) - len("_State") to a variable

Reply:
sure